buildbot/master.cfg

```diff
@@ -358,19 +358,19 @@ def make_dolphin_osx_universal_build(mode="normal"):
                            descriptionDone="mkbuilddir",
                            hideStepIf=StepWasSuccessful))
 
-    f.addStep(ShellCommand(command=["../BuildMacOSUniversalBinary.py", "-G", "Ninja", "--run_unit_tests"],
+    f.addStep(ShellCommand(command="security unlock-keychain -p $(cat ~/.keychain-password) buildbot.keychain-db",
```

Comment:
Does this create a `buildbot.keychain-db` file in the build directory? If so, we should probably add it to the gitignore to avoid version numbers being marked dirty -- or put the keychain file elsewhere.

Reply:
The keychain is stored in ``~/Library/Keychains`` (the security command automatically looks there). I pre-made this keychain in the GUI and loaded it with the ``Developer ID`` certificate and the notary credentials.